.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225350094-9f6e2e9d-2023-47dc-9c9d-1d811da63ed8.png)

**[Optional]**

- [ ] It would be nice if you make your branch name more descriptive :+1: because now you are using `staging` which is not descriptive enough. You could have used `feature/finalize-capstone` or `feature/finalize-good-jam` which is more descriptive and tells what you are doing in this branch :blush:

Reply:
Awesome. Big thanks for that insight, @HAMMAS-SALEEM 😊